static/js/components/inputs/SelectField.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import _ from 'lodash';
+
+import MenuItem from 'material-ui/MenuItem';
+import AutoComplete from '../AutoComplete';
+
+
+export default class SelectField extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
this constructor probably needs to be refactored. working on that (and linting errors) now